src/main/java/bookstore/controller/AuthController.java

```diff
@@ -0,0 +1,25 @@
+package bookstore.controller;
+
+import bookstore.dto.user.UserRegistrationRequestDto;
+import bookstore.dto.user.UserResponseDto;
+import bookstore.exception.RegistrationException;
+import bookstore.service.UserService;
+import jakarta.validation.Valid;
+import lombok.RequiredArgsConstructor;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RequiredArgsConstructor
+@RestController
+@RequestMapping("/api/auth")
+public class AuthController {
+    private final UserService userService;
+
+    @PostMapping("/registration")
```

Comment:
add swagger annotations

Reply:
Please, add a corresponding status code for POST method